packages/phrases/src/locales/en/errors/application.ts

```diff
@@ -8,7 +8,8 @@ const application = {
   user_consent_scopes_not_found: 'Invalid user consent scopes.',
   consent_management_api_scopes_not_allowed: 'Management API scopes are not allowed.',
   protected_app_metadata_is_required: 'Protected app metadata is required.',
-  protected_app_not_configured: 'Protected app provider is not configured.',
+  protected_app_not_configured:
```

Comment:
Where is this key used?

Reply:
You can search the code, when the protected app is not configured (system table)